File: src/main/java/com/ject/studytrip/mission/domain/repository/DailyMissionCommandRepository.java
package com.ject.studytrip.mission.domain.repository;

public interface DailyMissionCommandRepository {
    long deleteAllByDeletedAtIsNotNull();

    long deleteAllByDeletedMissionOwner();

    long deleteAllByDeletedDailyGoalOwner();

    long deleteAllByMemberId(Long memberId);
}

File: src/main/kotlin/com/ject/studytrip/mission/domain/repository/DailyMissionCommandRepository.kt
package com.ject.studytrip.mission.domain.repository

interface DailyMissionCommandRepository {
    fun deleteAllByDeletedAtIsNotNull(): Long

    fun deleteAllByDeletedMissionOwner(): Long

    fun deleteAllByDeletedDailyGoalOwner(): Long

    fun deleteAllByMemberId(memberId: Long): Long
}
